src/etc/roles/literal/noneRole.js

```diff
@@ -6,15 +6,39 @@ const noneRole: ARIARoleDefinition = {
   accessibleNameRequired: false,
   baseConcepts: [],
   childrenPresentational: false,
-  nameFrom: [],
-  prohibitedProps: [],
+  nameFrom: [
+    'prohibited',
+  ],
+  prohibitedProps: [
+    'aria-braillelabel',
+    'aria-label',
+    'aria-labelledby',
```

Comment:
also here

Reply:
Other than the `aria-braillelabel` which is the WAI-ARIA 1.3 additive change, this is a bugfix for the `none` role.

See related https://github.com/A11yance/aria-query/pull/551 which covers just the WAI-ARIA 1.2 bugfix.